ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Thanks for using Certbot - if you're experiencing issues with using Certbot
+we'd appreciate it if you could answer a few questions about your environment so
+that we can better assist you:
+
+## My operating system is (include version):
+
+
+## My web server is (include version):
+
+
+## How did you install Certbot:
+
+
+## What command did you run? What output did it produce?
+### Please provide the Certbot error log if you can. It is stored by default in `/var/log/letsencrypt`
+
+
+## Many problems can occur due to your server or DNS records being misconfigured. If you share the domain you're attempting to validate with us we can often debug the issue by accessing the URL and getting some information about this configuration. If you have concerns about publishing this domain publicly you contact one of the maintainers to share it over email.
```

Comment:
After thinking about this more, I don't think we should ask for this. I think we should restrict issues on this repo to be issues with Certbot and not more general problems configuring their server. I've found the domain is really useful for helping people debug a webserver they manually misconfigured with certificates from Certbot, but almost never needed to debug a problem with Certbot itself. I asked about this in IRC and everyone who responded agreed.

This coupled with the fact that people are very hesitant to give up their domain makes me think we should nix this question entirely.